pom.xml

```
@@ -16,7 +16,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <jenkins.version>2.60.3</jenkins.version>
+        <jenkins.version>2.289.1</jenkins.version>
         <java.level>8</java.level>
```

Removing java.level is required by `4.40`
```suggestion
        <jenkins.version>2.289.3</jenkins.version>
```

I suggest looking at https://github.com/jenkinsci/gitlab-branch-source-plugin/pull/189

---

Done in #9 so you just need to rebase.